pkg/app/template_event_handler.go

```diff
@@ -27,12 +28,14 @@ func (h *templateEventHandler) OnDelete(obj interface{}) {
 
 func (h *templateEventHandler) handle(obj interface{}) {
 	in := obj.(*smith.Template)
-	out := &smith.Template{}
 
-	if err := smith.DeepCopy_Template(in, out); err != nil {
+	c := conversion.NewCloner()
```

Comment:
Wait, can we pass an instance of Scheme into this handler and use it's Copy method?